mmv1/third_party/terraform/resources/resource_compute_instance.go

```diff
@@ -2239,3 +2240,28 @@ func hash256(raw string) (string, error) {
 	h := sha256.Sum256(decoded)
 	return base64.StdEncoding.EncodeToString(h[:]), nil
 }
+
+func serviceAccountDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
+	o, n := d.GetChange(strings.TrimSuffix(k, ".#"))
+	var l []interface{}
+	if old == "0" && new == "1" {
+		l = n.([]interface{})
+	} else if new == "0" && old == "1" {
+		l = o.([]interface{})
+	} else {
+		// we don't have one set and one unset, so don't suppress the diff
+		return false
+	}
+
+	// suppress changes between { } and {scopes:[]}
```

Comment:
Do we also need to check if the service account email changed?

It seems to me this will suppress diffs on the service account email even when there are valid changes.